card/lib/card/query/sql_statement.rb

```diff
@@ -49,7 +49,7 @@ def fields
       def full_field table, field
         case field
         when :raw      then "#{table}.*"
-        when :card     then "#{table}.name"
+        when :card     then "#{table}.*"
```

Comment:
This is, in some ways, the biggest change.  Instead of having the sql just return a name, it returns all the card info.  In cases where the cache is not populated, this means we can return 20 cards in 1 request as opposed to 21 (our current normal).   But, when the cache is populated, there is extra info getting passed from the database; I don't know how much of a cost that is.  But my notion is that as we hone this, we could make a smart guess about whether or not we're likely to need the extra fields by sticking a query key in the cache.  IF the query has expired, most likely some of the results have, too.